.github/dependabot.yml

```
@@ -47,6 +47,11 @@ updates:
     ignore:
       - dependency-name: "com.gradle:build-scan-plugin"
         versions: [ "1.16" ]
+  - package-ecosystem: "maven"
+    directory: "common-gradle-enterprise-maven-configuration"
```

did you mean this directory?
```suggestion
    directory: "gradle-enterprise-conventions-maven-extension"
```